.github/workflows/unit.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  go-tests:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 0.14.x
```

Comment:
Should update the terraform version used here to a more recent one

Reply:
have updated in other branch which will be merged before this